### ISSUE TITLE
Blademail detailed item description is incomplete 

### ISSUE THREAD

**Issue by ForestJohnson** [ValveSoftware/Dota2-Gameplay#7354]
### Hero and spell

Blademail

### Description

Blademail reflects the same damage type, so for example, if I activate blademail and then someone hits me with a basic attack (physical damage), then the blademail returns physical damage back to the attacker. 

This is true and matches the description on the item HOWEVER   in practice in the game it does not appear to be the case BECAUSE there is another un-documented blademail behavior: https://github.com/ValveSoftware/Dota2-Gameplay/issues/1872

Blademail always reflects the same damage type including physical damage EXCEPT when the attacker has BKB activated, then blademail does nothing at all.  

This "Physical blademail damage blocked by BKB" behaviour should be mentioned in the advanced info (hold down alt) inside the the tooltip. 

### Screenshots

![image](https://user-images.githubusercontent.com/7119703/223884155-5c2aa4fd-8088-462d-816e-1918be4ba2a4.png)

**Comment by Philaeux** [comment]
True. Advance tooltip is missing "Damage Return does not pierce spell immunity."

**Comment by ForestJohnson** [comment]
For clarity, consider 

> Damage Return does not pierce spell immunity, regardless of damage type.

**Comment by Philaeux** [comment]
Video is 6 years old, the behavior of blademail has changed since then. Namely:

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3007278/9862744e-8999-4125-9275-c294d7d7ac2d)
